### ISSUE TITLE
[Accessibility]: Congratulate users on getting help filling out a form

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35196]
### Audit type

Screen reader navigation

### User journey audit issue

#33517

### Problem description

<img width="640" alt="Screen Shot 2022-05-09 at 5 29 58 PM" src="https://user-images.githubusercontent.com/103087/167509485-e811ee5e-974e-4c84-82f3-1d0ef12f31be.png">

Hitting infer configuration from HEAD does not announce any change in state to the user. Inversely, when the button _becomes_ available on the screen it's also not announced. Its visibility can toggle depending on if the current text area state has changed from the inferred config or not.

### Expected behavior

I would expect that the entire contents of the text area on the page changing should be announced in some manner.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@efritz Going to assign this one to GitStart, LMK if you disagree/have already worked on a fix!

@gitstart-sourcegraph I think giving this `aria-live="polite"` would be a sufficient change to notify users

(sorry I posted this on the wrong issue before 😅 )

**Comment by umpox** [comment]
Reopening this! cc: @gitstart-sourcegraph
